components/__tests__/ProgressIndicator-test.js

```diff
@@ -59,26 +59,38 @@ describe('ProgressIndicator', () => {
     describe('ProgressStep', () => {
       it('should render with correct base className', () => {
         expect(
-          mountedList.find(ProgressStep).at(0).hasClass('bx--progress-step')
+          mountedList
+            .find(ProgressStep)
+            .at(0)
+            .hasClass('bx--progress-step')
         ).toEqual(true);
       });
 
       it('should render with a label', () => {
-        expect(mountedList.find(ProgressStep).at(0).props().label).toEqual(
-          'label'
-        );
+        expect(
+          mountedList
+            .find(ProgressStep)
+            .at(0)
+            .props().label
```

Comment:
Could also be:

```js
.props('label')
```

I think 😄

Reply:
I think it's supposed to be `.props().label`, here are the [enzyme docs](https://github.com/airbnb/enzyme/blob/master/docs/api/ReactWrapper/props.md)

Reply:
Sorry, I mistyped 😅 

http://airbnb.io/enzyme/docs/api/ShallowWrapper/prop.html

Reply:
e.g.

```diff
it('should render with a label', () => {
  expect(
    mountedList
      .find(ProgressStep)
      .at(0)
-     .props('label')
+     .prop('label')
  ).toEqual('label');
});
```

Reply:
Sorry, this took a while to update, made the changes in the latest commit.